src/utils/oauth2_util.py

```diff
@@ -3,6 +3,17 @@
 import bcrypt
 import jwt
 
+from src.exceptions.exceptions import UnAuthorizedEndpoint
+
+roles_endpoint_access = {
```

Comment:
Please map this using a class as key, for example:

```
roles_endpoint_access = {
   SecuredHttpRequestUrl(path='/chefs', method=MethodEnum.GET), [RoleEnum.ADMINISTRATOR]
}
```

Reply:
done.